reference/types/function.md

```diff
@@ -27,7 +27,9 @@ This exercise deals with cooking a lasagna dish in the oven. The reference imple
 | Track | Exercise                        | Changes |
 | ----- | ------------------------------- | ------- |
 | F#    | [basics][implementation-fsharp] | None    |
+| Julia | [functions-introduction][implementation-julia] | None |
```

Comment:
Unless I'm mistaken, the F# exercises teaches more than described above in this file, like defining a variable binding (not sure about the correct F# lingo). The Julia exercise has changes compared to the reference implementation, but matches the description of the reference implementation in this doc.

Reply:
Correct, but that is not relevant to this file. For this file, I only want to know what aspects of functions the F# exercise teaches. The assignment bits of the F# exercise are also in the `assignment.md` document. In other words, a single exercise can be in multiple documents (it covers multiple Concepts).

Reply:
You could consider also adding this exercise to the [assignment.md](https://github.com/exercism/v3/blob/master/reference/concepts/assignment.md) reference document.

Reply:
Okay, so because the functions-aspect of the exercise is the same as in the F# implementation, changes here would be `None` for Julia as well?

Reply:
assignment.md seems to be mostly about variable assignments, which aren't covered here.